example/device/framegrabber/grabRealSense2.cpp

```diff
@@ -90,9 +90,11 @@ int main()
 #if defined(VISP_HAVE_PCL) && defined(VISP_HAVE_THREADS)
     std::mutex mutex;
     pcl::PointCloud<pcl::PointXYZRGB>::Ptr pointcloud_color(new pcl::PointCloud<pcl::PointXYZRGB>());
```

Comment:
Is there `VISP_HAVE_PCL_COMMON` protection missing ?

```
#if defined(VISP_HAVE_PCL_COMMON)
pcl::PointCloud<pcl::PointXYZRGB>::Ptr pointcloud_color(new pcl::PointCloud<pcl::PointXYZRGB>());
#endif
```

Reply:
Done in commit 15914d275